api/apps/v1alpha1/nemo_datastore_types.go

```diff
@@ -88,20 +105,49 @@ type NemoDatastoreStatus struct {
 	State             string             `json:"state,omitempty"`
 }
 
-type NemoDataStoreParams struct {
-	AppVersion    string `json:"appVersion"`
-	GiteaEndpoint string `json:"giteaEndpoint"`
-	GiteaSecret   string `json:"giteaSecret"`
-	DatabaseURL   string `json:"databaseURL"`
-	DatabaseHost  string `json:"databaseHost"`
-	DatabasePort  string `json:"databasePort"`
-	DBSecret      string `json:"dbSecret"`
+type ObjectStoreConfig struct { // e.g. Minio, s3
```

Comment:
Shouldn't some of these fields under `ObjectStoreConfig`  be required?

Reply:
Ya they are required if there's the json tag does not have omitempty
See config/crd/bases/apps.nvidia.com_nemodatastores.yaml:L809-813

```
  required:
  - bucketName
  - credentials
  - endpoint
  - region
  - ssl
 ```